artemis-server/src/main/java/org/apache/activemq/artemis/core/server/cluster/ha/ReplicatedPolicy.java

```diff
@@ -61,6 +61,8 @@
 
    private final NetworkHealthCheck networkHealthCheck;
 
+   private int quorumVoteWait = ActiveMQDefaultConfiguration.getDefaultQuorumVoteWait();
```

Comment:
It would be better to just use the one provided by the constructor and declare this final if is not meant to be modified anymore